src/azure-cli/azure/cli/command_modules/servicebus/aaz/latest/servicebus/namespace/private_endpoint_connection/__cmd_group.py

```diff
@@ -0,0 +1,23 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command_group(
+    "servicebus namespace private-endpoint-connection",
+)
+class __CMDGroup(AAZCommandGroup):
+    """servicebus namespace private-endpoint-connection
+    """
```

Comment:
Maybe we can have some meaningful description for command group help msg?

Reply:
Done.